boards/shields/wnc_m14a2a/boards/nrf52840_pca10056.defconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52840_PCA10056
+
+config GPIO_NRF_P1
+	def_bool y
```

Comment:
The note at the end of https://docs.zephyrproject.org/latest/guides/kconfig/index.html#common-shorthands might apply here:

    For a symbol defined in multiple locations (e.g., in a Kconfig.defconfig
    file in Zephyr), it is best to only give the symbol type for the “base”
    definition of the symbol, and to use default (instead of def_<type> value)
    for the remaining definitions. That way, if the base definition of the
    symbol is removed, the symbol ends up without a type, which generates a
    warning that points to the other definitions. That makes the extra
    definitions easier to discover and remove.

Use `default` instead of `def_bool` if these already have a base definition.